src/platform_impl/macos/window.rs

```diff
@@ -464,8 +464,10 @@ impl UnownedWindow {
   }
 
   pub fn set_menu(&self, menu: Option<Menu>) {
-    unsafe {
-      util::set_menu_async(*self.ns_window, menu);
+    // TODO if None we should set an empty menu
+    // On windows we can remove it, in macOS we can't
```

Comment:
Is this still the case with the recent changes in PR #291 ?

Reply:
It still is. That PR isn't related.
I think to work this around we could create a menu with app name as an empty menu

Reply:
I thought that PR added the ability to manually hide/show the tray menu when the icon is clicked so theoretically if we call `set_menu(None)`  we should have the ability to not show it at all. Like #329 only shows the tray menu on right click without left click.

Reply:
#291 is about some mouse clicking misbehaviour iirc. But yeah I think we should make `set_menu(None)` not show the menu. On macOS, it will show a menu item with only application's name on it. We'll do this in future PR.